src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/__tests__/DynamicTableViewModelTests.swift

```diff
@@ -54,22 +54,20 @@ class DynamicTableViewModelTests: XCTestCase {
 		let section = sut.section(1)
 		
 		XCTAssertEqual(section.cells.count, 1)
-		let cell = section.cells.first
-		if case .bold(text: let text) = cell {
-			XCTAssertEqual(text, "Baz")
-		} else {
-			XCTFail()
-		}
+//		if let cell = section.cells.first {
```

Comment:
can this be removed completely?

Reply:
Well spotted. I missed that test. Will be updated shortly.